samples/go/go.mod

```diff
@@ -3,13 +3,14 @@ module github.com/miracl/oidc-samples/samples/go
 go 1.17
```

Comment:
Maybe update to latest as part of the maintenance.

Reply:
done